internal/lookout/ui/src/components/job-dialog/JobDetails.tsx

```diff
@@ -59,6 +74,38 @@ export default function JobDetails(props: DetailsProps) {
             <DetailRow name="Job state" value={props.job.jobState} />
             <DetailRow name="Priority" value={props.job.priority.toString()} />
             <DetailRow name="Submitted" value={props.job.submissionTime} />
+            {command &&
+              command.map(
```

Comment:
Rather than displaying everything in the same table, I would have a separate expandable list of containers, with conatiner name, command, args, and requests for each container.

In this way you could have a function to get a list of Containers, instead of having many functions to get list of cpu requests, list of memory requests, etc... individually

Reply:
Good suggestion.  

I am not sure what you mean by "expandable list of containers".  Do you mean that this should be a react component of a List?  

I tried to combine all those separate functions into a single function so the usability of getting the container info is easy for the UI.  

Let me know if this is what you want.